pacman/utilities/utility_calls.py

```diff
@@ -15,21 +15,7 @@
 
 import hashlib
 import numpy
-from pacman.exceptions import (
-    PacmanInvalidParameterException, PacmanValueError)
-
-
-def locate_constraints_of_type(constraints, constraint_type):
```

Comment:
It doesn't look like locate_first_constraint_of_type(..) (the next function in this file) is used anywhere either so this can probably be deleted.